pkg/kubelet/apis/cri/runtime/v1alpha3/api.proto

```diff
@@ -91,6 +91,9 @@ service RuntimeService {
 
     // Status returns the status of the runtime.
     rpc Status(StatusRequest) returns (StatusResponse) {}
+
+    //  GetRuntimeConfigInfo returns the configuration details of the runtime.
+    rpc GetRuntimeConfigInfo(GetRuntimeConfigInfoRequest) returns (GetRuntimeConfigInfoResponse) {}
```

Comment:
Could this be part of `Status()`?

Reply:
`Status` right now is used primarily to check the health of the runtime, and is called periodically. Using a separate call is cleaner and may avoid hitting performance problems.